code-of-conduct-proposed.md

```diff
@@ -52,23 +54,26 @@ Examples of behavior that contributes to a positive environment include:
   and learning from the experience
 * Focusing on what is best not just for us as individuals, but for the
   overall community
+* Using welcoming and inclusive language
```

Comment:
Agreed that using welcoming and inclusive language contributes to a positive environment. 

Note that not everyone really understands how to use inclusive language, so I would not want anyone to be punished for unintentionally using a term that is exclusionary (e.g., using the term "grandfathered" if they don't understand its racist origins). So I would not be supportive of including failure to use inclusive language to the "unacceptable behavior" list, but I am supportive of this change as an example of positive behavior

Reply:
I totally agree with your point. Always promoting positive behavior is more inclusive.